<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PWMSearchDialog</class>
 <widget class="QDialog" name="PWMSearchDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>615</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>536</width>
    <height>575</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Weight Matrix Search</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="sizeConstraint">
    <enum>QLayout::SetMinAndMaxSize</enum>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Matrix:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="modelFileEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QToolButton" name="pbSelectModelFile">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="scoreLabel">
       <property name="toolTip">
        <string>Minimal score</string>
       </property>
       <property name="text">
        <string>Score:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSlider" name="scoreSlider">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>85</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksBelow</enum>
       </property>
       <property name="tickInterval">
        <number>5</number>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="scoreValueLabel">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>85%</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="jasparButton">
       <property name="text">
        <string>Search JASPAR Database</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buildButton">
       <property name="text">
        <string>Build new matrix</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="viewMatrixButton">
       <property name="text">
        <string>View matrix</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1">
     <item>
      <widget class="QLabel" name="algorithmLabel">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Weight algorithm</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="algorithmCombo">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="0,1">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="bgStrand">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Strands</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QRadioButton" name="rbBoth">
            <property name="text">
             <string>Both strands</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rbDirect">
            <property name="toolTip">
             <string/>
            </property>
            <property name="whatsThis">
             <string>Direct strand</string>
            </property>
            <property name="shortcut">
             <string/>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rbComplement">
            <property name="toolTip">
             <string/>
            </property>
            <property name="whatsThis">
             <string>Reverse complement strand</string>
            </property>
            <property name="shortcut">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="rangeSelectorLayout"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinAndMaxSize</enum>
     </property>
     <item>
      <widget class="QTreeWidget" name="tasksTree">
       <property name="rootIsDecorated">
        <bool>false</bool>
       </property>
       <property name="itemsExpandable">
        <bool>true</bool>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <column>
        <property name="text">
         <string>Matrix</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Minimal score</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Algorithm</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="sizeConstraint">
        <enum>QLayout::SetMinAndMaxSize</enum>
       </property>
       <item>
        <widget class="QPushButton" name="loadListButton">
         <property name="text">
          <string>Load list</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="saveListButton">
         <property name="text">
          <string>Save list</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="loadFolderButton">
         <property name="text">
          <string>Load folder</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="clearListButton">
         <property name="text">
          <string>Clear list</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeWidget" name="resultsTree">
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <property name="columnCount">
      <number>4</number>
     </property>
     <column>
      <property name="text">
       <string>Range</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Matrix</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Strand</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Score</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="pbClear">
       <property name="text">
        <string>Clear results</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbSaveAnnotations">
       <property name="text">
        <string>Save as annotations</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="statusLabel">
       <property name="text">
        <string>Status</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::Yes</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
